.github/workflows/CI.yaml

```
@@ -29,9 +29,10 @@ jobs:
       uses: actions/setup-python@v5
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
+    - uses: glotzerlab/workflows/setup-uv@ea2e25d07af862a1c696a932c2bd6b242d142049
```

```suggestion
    - uses: glotzerlab/workflows/setup-uv@ea2e25d07af862a1c696a932c2bd6b242d142049 # 0.2.0
      with:
        lockfile: ".github/requirements${{matrix.python-version}}.txt"
```
And delete the now duplicate "Set up Python environment" step.

---

This doesn't work in its current implementation because miniball doesn't have binaries. See [glotzerlab/workflows#8](https://github.com/glotzerlab/workflows/issues/8)

---

```suggestion
    - uses: glotzerlab/workflows/setup-uv@ea2e25d07af862a1c696a932c2bd6b242d142049 # 0.2.0
      with:
      	onlybinary: "false"
        lockfile: ".github/requirements${{matrix.python-version}}.txt"
```